README.md

```diff
@@ -33,7 +33,6 @@ logger.info('Test logging', { exampleTag: 'additional details' });
 // This will call `console.log` with `{'message': 'Test logging', 'exampleTag': 'additional details'}`
 ```
 
-
```

Comment:
No objection to removal, but just mentioning that this "double new line" before header style is intentional. I do it to add more visual distinction to different sections.

Reply:
Hmm, I don't think I removed this intentionally so it must have been prettier? It appears we have `lint-staged` setup to prettify `*.md` so I'm not sure why it didn't get changed before. Maybe this `README` predates the `lint-staged` setup?